core/src/test/java/io/questdb/griffin/OrderByAdviceTest.java

```diff
@@ -1186,4 +1186,18 @@ public void testDistinctWithOrderByAnotherColumn() throws Exception {
             TestUtils.assertContains(e.getFlyweightMessage(), "ORDER BY expressions must appear in select list.");
         }
     }
+
```

Comment:
It would be good to also  test it with :
- order by column_position
- column aliases 
- different column name case and/or quoted identifiers